programs/mango-v4/tests/program_test/mango_client.rs

```diff
@@ -1283,7 +1284,64 @@ impl ClientInstruction for TokenMakeReduceOnly {
         let mint_info: MintInfo = account_loader.load(&mint_info_key).await.unwrap();
 
         let instruction = Self::Instruction {
-            reduce_only_opt: Some(true),
+            reduce_only_opt: Some(1),
+            ..token_edit_instruction_default()
+        };
+
+        let accounts = Self::Accounts {
+            group: self.group,
+            admin: self.admin.pubkey(),
+            mint_info: mint_info_key,
+            oracle: mint_info.oracle,
+        };
+
+        let mut instruction = make_instruction(program_id, &accounts, instruction);
+        instruction
+            .accounts
+            .extend(mint_info.banks().iter().map(|&k| AccountMeta {
+                pubkey: k,
+                is_signer: false,
+                is_writable: true,
+            }));
+        (accounts, instruction)
+    }
+
+    fn signers(&self) -> Vec<TestKeypair> {
+        vec![self.admin]
+    }
+}
+
+pub struct TokenMakeForceClose {
```

Comment:
personally, I'd have joined this with TokenMakeReduceOnly since it's only used rarely and the two parameters are related...